docs/rules/0121/resource-must-support-get.md

```diff
@@ -0,0 +1,79 @@
+---
+rule:
+  aip: 121
+  name: [core, '0121', resource-must-support-get]
+  summary: All resource names must use camel case in collection identifiers.
+permalink: /121/resource-must-support-get
+redirect_from:
+  - /0121/resource-must-support-get
+---
+
+# Resource must support get
+
+This rule enforces that all resources support the Get operation as mandated in
+[AIP-121][].
+
+## Details
+
+This rule scans a service for Create, Update, and List methods for resources,
+and ensures each one has a Get method.
+
+## Examples
+
+**Incorrect** code for this rule:
+
+```proto
+// Incorrect.
+service Foo {
+  // Book has a create, but no Get method.
+  rpc CreateBook(CreateBookRequest) returns (Book) {};
+}
+
+message Book {
+	option (google.api.resource) = {
+		type: "library.googleapis.com/Book"
+	};
+}
+```
+
+**Correct** code for this rule:
+
+```proto
+// Correct.
+service Foo {
+  rpc CreateBook(CreateBookRequest) returns (Book) {};
```

Comment:
Nit: spacing

Reply:
sorry :( Not sure what's happening there.. I'll look at my formatter.